angry-bot/src/data/feet.ts

```diff
@@ -11,14 +11,51 @@ export const feetRelated = [
     "👡",
     "👢",
     "🥿",
+    "⛸️",
+    "🧦",
+    "🩴",
+    "🩰",
+    "🤸‍♀️",
+    "🤸‍♂️",
+    "🤸",
     "fuß",
     "feet",
     "toe",
     "zeh",
+    "zehen",
+    "füße",
     "fuss",
+    "Bein",
+    "Knie",
+    "Sohle",
+    "Fersen",
+    "Fersensporn",
+    "Hallux",
+    "Pediküre",
+    "Podologie",
+    "Schuh",
+    "Sneakers",
+    "Tritt",
+    "Zehennagel",
+    "Zehennagelpilz",
+    "Zehnägel",
+    "Zehring",
+    "Zehverletzung",
```

Comment:
des sollte alles lowercase sein, weil de message wird mit `.toLowerCase()` mit dem verglichen.
und i glaub alles mit "zeh" drinnen is scho von "zeh" abgedeckt